### PR TITLE
Dev 31194

### PR DESCRIPTION
**Pull request by acroaa**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -604,6 +614,12 @@ export interface AcrolinxSidebar {
    * @param message The message to show.
    */
   showMessage(message: Message): void;
+
+  /**
+   * Perform a search for suggestions with the given prefix .
+   * @param prefix The prefix on which to perform a search for suggestions.
+   */
+  reusePrefixSearch?(prefix: string): void;
```
Comment (JustinHorn) [inline src/index.ts:622]: Since it is not a prefix search anymore, we should name this function differently. I would call it "startReuseSearch"

**Review by JustinHorn — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 2]
```diff
@@ -706,6 +731,29 @@ export interface AcrolinxPlugin {
...
+  /**
+  * the user has clicked on the button to open reuse panel. The AcrolinxPlugin is requested to open the reuse panel.
+  */
+  openReusePanel?(): void;
+
+  /**
+  * Notifies the AcrolinxPlugin that the user has changed the UI langauge in the sidebar.
+  * @param uiLanguage   The selected UI language
+  */
+  onUiLanguageChanged?(uiLanguage: UILanguage): void;
```
Comment (JustinHorn) [inline src/index.ts:756]: this should be onU**I**LanguageChanged

File: src/index.ts (hunk 1 above)
Comment (acroaa) [reply src/index.ts]: Updated to "liveSearch"

File: src/index.ts (hunk 2 above)
Comment (acroaa) [reply src/index.ts]: @JustinHorn Updated. Please make sure to also update it on your side

**Review by loaiabdelhalim — APPROVED** [review]

**Pull request merged**